software/containers_on_frontier.rst

```diff
@@ -122,12 +122,68 @@ This example will look at building a container image for testing collective comm
       Bootstrap: oras
       From: docker.io/subilabrahamornl/opensusempich342rocm571:latest
 
+OLCF Base Images & Apptainer Modules
+-------------------------------------
+To assist the container workflow on Frontier, OLCF provides some base container images and apptainer modules to
+simplify the process. The following sections document them and provide an `Example Workflow`_.
 
+Base Images
+^^^^^^^^^^^
+
+Due to copyright issues, OLCF is not able to provide containers with the Cray Programming Environment (CPE) installed
```

Comment:
I would reword this to something like: "Due to licensing, OLCF is currently not able to provide containers with the Cray Programming Environment (CPE) installed in them"

Reply:
Fixed in [b1069b3](https://github.com/olcf/olcf-user-docs/pull/870/commits/b1069b3a71210e8864cc6eeb20f70d7e5c8e53e7)